docs/plugins/using-toolkits.md

```diff
@@ -73,19 +73,51 @@ my-profile:
 
 ## Starting a Goose Session with Toolkits
 
-Once your [profile](https://block.github.io/goose/guidance/getting-started.html#configuring-goose-with-the-profilesyaml-file) is set up, start a Goose session with the specified profile:
+Once your [profile](https://block.github.io/goose/guidance/getting-started.html#configuring-goose-with-the-profilesyaml-file) is set up, you can start a Goose session with the specified profile. You can configure multiple profiles in your profiles.yaml file to suit different needs. For example, you might have separate `default` and `work` profiles:
+
+```yaml
+default:
+  provider: openai
+  processor: gpt-4o
+  accelerator: gpt-4o-mini
+  moderator: synopsis
+  toolkits:
+  - name: synopsis
+    requires: {}
+  - name: screen
+    requires: {}
+work:
+  provider: block
+  processor: gpt-4o
+  accelerator: gpt-4o-mini
+  moderator: synopsis
+  toolkits:
+  - name: synopsis
+    requires: {}
+  - name: jira
+    requires: {}
+```
+
+In the example above:
+- The `default` profile uses the `synopsis`, and `screen` toolkits.
+- The `work` profile is configured with the `synopsis` and `jira` toolkits.
```

Comment:
1. is it ok for these to both be in one file?
2. can we add the file name to the code snippet?

Reply:
1. yes it goes in the same file 
2. yes